components/Wallet/Wallet.tsx

```diff
@@ -27,8 +26,7 @@ export function Wallet() {
   }, [connectedWallets, wallet]);
 
   useEffect(() => {
-    if (!connect) return;
-
+    if (!connect || connectedWallets?.length > 0) return;
```

Comment:
this was a bug where it would show an "action required in metamask" popup on navigation.